### PR TITLE
License and podspec

### PR DESCRIPTION
**Pull request by jacksonh**
Adds the MIT license file and a podspec which I will submit to the main specs repo if you can tag the repo.

### CODE REVIEW

File: CFIFrostedOverlayView.podspec [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Pod::Spec.new do |s|
+  s.name         = "CFIFrostedOverlayView"
+  s.version      = "0.0.1"
```
Comment (CodaFi) [inline CFIFrostedOverlayView.podspec:3]: Doesn't this require that I tag 0.1.1 first?

File: CFIFrostedOverlayView.podspec [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  		   A view that renders its superview with a gaussian blur like iOS 7's Control Center.
+                   DESC
+
+  s.homepage     = "https://github.com/CodaFi/CFIFrostedOverlayView"
+  s.screenshots  = "https://raw.github.com/CodaFi/CFIFrostedOverlayView/master/Artwork/Screen%20Shot%202013-06-16%20at%201.50.35%20PM.png"
+
+  s.license      = { :type => 'MIT', :file => 'LICENSE' }
+  s.author       = { "Robert Widmann" => "devteam.codafi@gmail.com" }
+
+  s.ios.deployment_target = '7.0'
```
Comment (CodaFi) [inline CFIFrostedOverlayView.podspec:16]: This deploys back to iOS 5 and whatever GPUImage supports for the Mac.

File: CFIFrostedOverlayView.podspec (hunk 1 above)
Comment (jacksonh) [inline CFIFrostedOverlayView.podspec:3]: My normal workflow is to check in the podspec, tag, and then submit the podspec to the main specs repo. That does leave things in an invalid state temporarily though.

File: CFIFrostedOverlayView.podspec (hunk 2 above)
Comment (jacksonh) [inline CFIFrostedOverlayView.podspec:16]: Any idea which version of OSX I should target?

**Comment by jacksonh** [comment]
Probably time to close this out

**Pull request closed without merging**